app/Resources/meta/imageRendition.yml

```diff
@@ -11,4 +11,7 @@ properties:
     height:
         description: "Rendition height"
         type: int
+    convertedToWebp:
```

Comment:
if it's a boolean value, then the `isConvertedToWebp` sounds much better

Reply:
but then getter used by Meta class would be `isIsConvertedToWebp`. I will need to add some more logic to meta class getters generator.